packages/kbn-user-profile-components/src/user_profiles_selectable.tsx

```diff
@@ -0,0 +1,264 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import type { EuiSelectableOption, EuiSelectableProps } from '@elastic/eui';
```

Comment:
If the default messages are oriented around user profiles I do not think we need any customization. For example, the "No matching users with required access" looks good to me. @jonathan-buttner what do you think?

Reply:
I think Oleg is correct here in that it's probably best to build this component without a specific use-case in mind. 

I will change the copy to use the term "user" instead of "assignee" but expose the following props so that it can be overridden:

- loadingMessage
- noMatchesMessage
- emptyMessage
- errorMessage
- searchPlaceholder
- selectedStatusMessage
- clearButtonLabel

Reply:
Yeah the message you had works for our use case. But we can leverage the props your exposing 👍